src/main/java/mate/academy/intro/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.intro.controller;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.intro.dto.CartItemRequestDto;
+import mate.academy.intro.dto.ShoppingCartDto;
+import mate.academy.intro.service.ShoppingCartService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/cart ")
```

Comment:
add `server.servlet.context-path=/api` to application.properties
```suggestion
@RequestMapping("/cart")
```